Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.yml

```diff
@@ -0,0 +1,191 @@
+category: Forensics & Malware Analysis
+commonfields:
+  id: WildFire-Reports
+  version: -1
+configuration:
+- defaultvalue: https://wildfire.paloaltonetworks.com/publicapi
+  display: Server base URL (e.g. https://192.168.0.1/publicapi)
+  name: server
+  required: true
+  type: 0
+- display: API Key
+  name: token
+  required: true
+  type: 4
+- additionalinfo: Reliability of the source providing the intelligence data.
+  defaultvalue: B - Usually reliable
+  display: Source Reliability
+  name: integrationReliability
+  options:
+  - A+ - 3rd party enrichment
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Return warning entry for unsupported file types
+  name: suppress_file_type_error
+  required: false
+  type: 8
```

Comment:
please always supress, so the widget will not present an error, and rm this param.